lib/services/nativescript-project-plugins-service.ts

```diff
@@ -48,7 +49,16 @@ export class NativeScriptProjectPluginsService implements IPluginsService {
 	public findPlugins(keywords: string[]): IFuture<IBasicPluginInformation[]> {
 		return ((): IBasicPluginInformation[] => {
 			let findPluginByNameFutures = _.map(keywords, keyword => this.findPluginsByName(keyword));
-			return getFuturesResults<IBasicPluginInformation>(findPluginByNameFutures, plugins => !!plugins && !!plugins.length);
+			Future.wait(findPluginByNameFutures);
+			let findPluginsResults: IBasicPluginInformation[][] = findPluginByNameFutures.map(f => f.get()).filter(plugins => !!plugins && !!plugins.length);
+
+			let pluginsNamesInGroups = findPluginsResults.map(res => _.map(res, r => r.name));
+			let matchingNames = _.intersection.apply(this, pluginsNamesInGroups);
```

Comment:
nope, `intersection` works with many arrays passed as separate parameters, pluginsNamesInGroups is array of arrays. That's what the apply is doing for me - unpacking the array of arrays to different arguments, each of them is array. This way intersection works correctly